### ISSUE TITLE
Open in new window / tab icon on link doesn't turn purple when visited in Safari

### ISSUE THREAD

**Issue by GCHQ-Developer-847** [mi6/ic-ui-kit#2197]
## Summary of the bug 
The 'open in new window / tab' icon which can be displayed next to a link does not purple when the link is visible in Safari.

## 🪜 How to reproduce 
1. On Safari, find an external link e.g. [on the website](https://design.sis.gov.uk/accessibility) (at the bottom of the page).
2. Click on the link (to make it become a visited link).
3. Go back to the link and see that the text has turned purple but the icon is still blue.

## 📸 Screenshots or code 
<img src="https://github.com/user-attachments/assets/dfe15cfa-cbf8-4859-bfd5-30b8b7017ce6" width="300">

## 🖥 📱 Device 
- Browser: Safari

## 🧐 Expected behaviour 
The icon should turn purple, matching the text.

**Comment by evenstensberg** [comment]
MDX/Gatsby issue so fixed it there: https://github.com/mi6/ic-design-system/pull/1382

**Comment by MI6-255** [comment]
Moved out of done as was causing an issue with links on Hero appearing as the purple visited colour. Changes have been reverted.

**Comment by GCHQ-Developer-530** [comment]
This is a [known issue with Safari 15](https://bugs.webkit.org/show_bug.cgi?id=244025), it has been resolved so should be resolved for us without any changes in a future Safari update.
